app/services/bulk_upload/sales/year2023/row_parser.rb

```diff
@@ -816,6 +816,9 @@ def attributes_for_log
     attributes["mortgageused"] = mortgageused
 
     attributes["uprn"] = field_19
+    attributes["uprn_known"] = 1 if field_19.present?
```

Comment:
this will conflict with the PR I've just put in, I set uprn_known as 0 if field_19 is not present as well.  Just a heads up in case mine gets in first :)

Reply:
this one: (https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1625)

Reply:
Yup, sounds good, I'll rebase when you get yours merged, thanks!